### PR TITLE
DOC: doc fix

### PR DESCRIPTION
**Pull request by simonjayhawkins**
https://github.com/pandas-dev/pandas/issues/35831#issuecomment-688518465

### CODE REVIEW

**Comment by jbrockmendel** [comment]
LGTM

File: doc/source/whatsnew/v1.1.2.rst [hunk 1]
```diff
@@ -51,7 +51,7 @@ Bug fixes
 Other
 ~~~~~
 - :meth:`factorize` now supports ``na_sentinel=None`` to include NaN in the uniques of the values and remove ``dropna`` keyword which was unintentionally exposed to public facing API in 1.1 version from :meth:`factorize` (:issue:`35667`)
-- :meth:`DataFrame.plot` and meth:`Series.plot` raise ``UserWarning`` about usage of FixedFormatter and FixedLocator (:issue:`35684` and :issue:`35945`)
+- :meth:`DataFrame.plot` and :meth:`Series.plot` raise ``UserWarning`` about usage of FixedFormatter and FixedLocator (:issue:`35684` and :issue:`35945`)
```
Comment (dsaxton) [inline doc/source/whatsnew/v1.1.2.rst:54]: Should FixedFormatter and FixedLocator have double backticks?

Comment (jreback) [reply doc/source/whatsnew/v1.1.2.rst:54]: too bad the docs build doesn't warn on these things. maybe can have some sort of code check (at least to check the book-ended meth/func and so on

Comment (simonjayhawkins) [reply doc/source/whatsnew/v1.1.2.rst:54]: > Should FixedFormatter and FixedLocator have double backticks?

not sure. they are mpl classes. added anyway.

may have been better to include the message verbatim in this case instead `UserWarning: FixedFormatter should only be used together with FixedLocator` as was done for `ValueError: buffer source array is read-only` above.

**Pull request merged**